src/webapp/cypress/integration/admin/menu.js

```diff
@@ -57,35 +55,5 @@ sizes.forEach(size => {
                 expect(response.status).to.eq(403)
             })
         })
-
-        it('should allow an admin to impersonate another user', () => {
```

Comment:
Why is this away?  Should it not be replaced?

Reply:
This is a duplicate of a test case in impersonate.js in the same folder.